docs/Users_Guide/appendixA.rst

```diff
@@ -12,1720 +12,1796 @@ File-IO
-plotting features. However, the ASCII output statistics of MET may be plotted
-with a wide variety of plotting packages, including R, NCL, IDL, and GNUPlot.
-METViewer is also currently being developed and used by the DTC and NOAA
-It creates basic plots of MET output verification statistics. The types of
-plots include series plots with confidence intervals, box plots, x-y scatter
-plots and histograms.
-
-R is a language and environment for statistical computing and graphics.
-It's a free package that runs on most operating systems and provides nice
-plotting features and a wide array of powerful statistical analysis tools.
-There are sample scripts on the
-`MET website <http://dtcenter.org/community-code/model-evaluation-tools-met/sample-analysis-scripts>`_
-that you can use and modify to perform the type of analysis you need.  If
-you create your own scripts, we encourage you to submit them to us through the
-`METplus GitHub Discussions Forum <https://github.com/dtcenter/METplus/discussions>`_
-so that we can post them for other users. 
+  .. dropdown:: Answer
+		
+     MET provides some :ref:`plotting and graphics support<plotting>`.
+     The plotting
+     tools, including plot_point_obs, plot_data_plane, and plot_mode_field, can
+     help users visualize the data. 
+
+     MET is intended to be a set of command line tools for evaluating forecast
+     quality. So, the development effort is focused on providing the latest,
+     state of the art verification approaches, rather than on providing nice
+     plotting features. However, the ASCII output statistics of MET may
+     be plotted
+     with a wide variety of plotting packages, including R, NCL, IDL,
+     and GNUPlot.
+     METViewer is also currently being developed and used by the DTC and NOAA
+     It creates basic plots of MET output verification statistics. The types of
+     plots include series plots with confidence intervals, box plots,
+     x-y scatter plots and histograms.
+
+     R is a language and environment for statistical computing and graphics.
+     It's a free package that runs on most operating systems and provides nice
+     plotting features and a wide array of powerful statistical analysis tools.
+     There are sample scripts on the
+     `MET website <http://dtcenter.org/community-code/model-evaluation-tools-met/sample-analysis-scripts>`_
+     that you can use and modify to perform the type of analysis you need.  If
+     you create your own scripts, we encourage you to submit them to us
+     through the
+     `METplus GitHub Discussions Forum <https://github.com/dtcenter/METplus/discussions>`_
+     so that we can post them for other users. 
 
 Q. How do I find the version of the tool I am using?
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    
-A.
-Type the name of the tool followed by **-version**. For example,
-type “pb2nc **-version**”.
+  .. dropdown:: Answer
+		
+     Type the name of the tool followed by **-version**. For example,
```

Comment:
```suggestion
     Type the name of the tool followed by **--version**. For example,
```
This should be two dashes instead of one.  Please take a look in RTD to make sure it picks up the additional dash.